mypy/hooks.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, Optional, Callable, Union
+from mypy.types import Type
+
+hooks = {}  # type: Dict[str, Callable]
+
+docstring_parser_type = Callable[[str, int], Optional[Dict[str, Union[str, Type]]]]
```

Comment:
A type alias like this should have a CapWords name. I would also prefer this to always return a string.